### ISSUE TITLE
Cannot pause a HA media player

### ISSUE THREAD

**Issue by robimac04** [music-assistant/hass-music-assistant#2262]
### What version of Music Assistant has the issue?

2.0.2

### What version of the Home Assistant Integration have you got installed?

2024.5.1

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

I cannot pause media when using the `Home Assistant MediaPlayers` provider. I get the following error:

<img width="1158" alt="image" src="https://github.com/music-assistant/hass-music-assistant/assets/19255328/e156004f-4379-4c70-b506-6c2e913e1fc9">

Turning off the media player from the player selection menu also doesn't work (same error). However, skipping works fine, as well as pausing and turning off the player from Home Assistant itself. 

### How to reproduce

1. Play media with a `Home Assistant MediaPlayers` device
2. Try to pause the music through the pause button or turning off the device

### Music Providers

YouTube Music

### Player Providers

Home Assistant MediaPlayers

### Full log output

[mass.log](https://github.com/music-assistant/hass-music-assistant/files/15286014/mass.log)

### What version of Home Assistant Core are your running

2024.5.3

### What type of installation are you running?

Home Assistant Supervised

### On what type of hardware are you running?

Raspberry Pi

**Comment by OzGav** [comment]
What would be great is if you could provide the info requested by ou troubleshooting docs which you ticked that you have read. Such as:

What music provider is in use when the problem is observed? What other music providers have you tried (if you have no others then make that clear)
What player provider is in use when the problem is observed? What other player providers have you tried (if you have no others then make that clear)
How is playback being instigated (e.g. automation or via the UI)
What is working (e.g. player works with music provider X but not Y)
What you have tried from this page that hasn't helped

**Comment by robimac04** [comment]
Thanks for your reply. 
As mentioned in the original report, I'm using `YouTube Music` as music provider and `Home Assistant MediaPlayers` as player provider. Haven't tried any other providers, means I cannot tell if this happens with other music or player providers.

- Playback and commands are initiated via UI.
- No complex network setup for the speakers: Home Assistant and speakers are in the same VLAN and can communicate directly
- Playing media itself works fine
- Speakers are not grouped

**Comment by OzGav** [comment]
Can you try other providers? We ideally want to know if this is specifically a problem with one provider or is a core issue affecting all providers

What do you mean "Playing media itself works fine"

**Comment by robimac04** [comment]
>What do you mean "Playing media itself works fine"  

I can click the Play button for media (single tracks, playlists, radio stations) and it starts playing successfully, so that I can hear it from my speaker. Means, there is no connectivity issue between HA and the speakers which may block playback all together.

>Can you try other providers?

I've just tried DLNA now, there I can pause music. So this seems to be an issue with `Home Assistant MediaPlayers` player provider.

Just for reference, the speaker I'm using is a Bose SoundTouch 10, integrated into Home Assistant via Bose SoundTouch integration.

**Comment by OzGav** [comment]
By play button you mean the play button on the HA entity I assume.

Trying this myself I don't see your problem but I see other issues. This will get looked at in due course.

**Comment by jurgenhaas** [comment]
I'm seeing a similar issue which wasn't a problem before the 2.0.0 release but is now. In my case it's a chromecast device together with a radio stream from Tune-In.

Pausing the stream from within HA does work, but I can't mute. The HA interface shows an error message `Failed to call service media_player/volume_mute. Entity media_player.wohnzimmer_audio does not support this service.`

**Comment by OzGav** [comment]
@robimac04 Can you also try DIFFERENT HA players and see if this affects all the HA media players you have or maybe just a specific type.

**Comment by robimac04** [comment]
I just updated to MA 2.0.3 and this changed some things. Now I can play music from the MA interface, but it isn't showing up as playing in the lower bar at all:

<img width="1545" alt="image" src="https://github.com/music-assistant/hass-music-assistant/assets/19255328/5fdd3cd9-c2db-48ab-a0f3-1e9fcd9e9dad">

It looks exactly like this, but the music is playing. If I click on the play button in the lower bar (picture), the song starts over. I can however turn off the speaker now through the speaker selection in the sidebar now (before the update, this presented me the same error as with pausing as described above). 

This all happened with the same speaker entity as before (SoundTouch 10). But I've also tried that with my HomePod mini (Apple TV integration in HA), exactly the same behavior:

<img width="1544" alt="image" src="https://github.com/music-assistant/hass-music-assistant/assets/19255328/1aebe610-108b-4d57-8708-729185acc5cf">

**Comment by OzGav** [comment]
@robimac04 please don’t hijack this report with unrelated questions. Open your own report after reviewing the troubleshooting page. I will delete this and your post soon.

**Comment by OzGav** [comment]
Its getting hard to follow what the issue is. This started as being unable to PAUSE a HA MEDIA PLAYER. Has that been resolved?

**Comment by OzGav** [comment]
We will close this soon due to no follow up

**Comment by robimac04** [comment]
Sorry for the delay. Yes, still an issue.

**Comment by OzGav** [comment]
@robimac04 While this gets looked at further just wanted to check that you are normally using the MA Airplay provider for the Homepod Mini?

**Comment by robimac04** [comment]
@OzGav Can confirm, media controls work as intended when using the MA Airplay provider for the HomePod mini.

**Comment by OzGav** [comment]
@robimac04 Can you try with 2.1.0rc3?

**Comment by robimac04** [comment]
@OzGav Sure, in 2.1.0rc3 it works as expected 👍
